template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/README.md.tmpl

```diff
@@ -237,10 +237,21 @@ i.e. for each environment
 You can run unit tests for your ML code via `pytest tests`.
 
 {{ end }}
+
 ## Next Steps
-When you're satisfied with initial ML experimentation (e.g. validated that a model with reasonable performance can be
-trained on your dataset) and ready to deploy production training/inference
-pipelines, ask your ops team to set up CI/CD for the current ML project if they haven't already. CI/CD can be set up as part of the 
-MLOps Stacks initialization even if it was skipped in this case, or this project can be added to a repo setup with CI/CD already,
-following the directions under "Setting up CI/CD" in the repo root directory README.
+
+When you're satisfied with initial ML experimentation (e.g. validated that a model with reasonable performance can be trained on your dataset) and ready to deploy production training/inference
+pipelines, ask your ops team to set up CI/CD for the current ML project if they haven't already. CI/CD can be set up as part of the
+
+MLOps Stacks initialization even if it was skipped in this case, or this project can be added to a repo setup with CI/CD already, following the directions under "Setting up CI/CD" in the repo root directory README.
+
+To add CI/CD to this repo:
+ 1.  Run `databricks bundle init mlops-stacks` in your home directory
+ 2. Select the option to only initialize `CICD_Only`
+ 3. Provide the current root directory and answer the subsequent prompts
+
+If you already have CI/CD setup in a separate:
+1. Create a new folder in your root CI/CD repo
+1. Copy the contents of your current project folder into the new folder
```

Comment:
Bit confused on what these instructions aim to accomplish? If they have random CI/CD files, they'll need to do some reworking to get them to work with this project

Reply:
Isnt there a user journey, where the user can copy paste their project only repo into an already existing CI/CD only repo?